docs/pages/image-extractor.rst

```diff
@@ -145,23 +147,94 @@ and DataLoader, refer to this guide: https://pytorch.org/tutorials/beginner/data
 
     for i, sample_batch in enumerate(dataloader):
         img, label = sample_batch["rgba"], sample_batch["label"]
-        img = img.permute(0, 3, 1, 2)  # Reshape to PyTorch format for convolutions
+        img = img.permute(0, 3, 1, 2)
         out = net(img)
 
 
+`Writing Custom Pose Extractors`_
+=================================
+
+The pose extractor defines how camera poses are programmatically determined so that the image
+extractor knows how to manipulated the camera position and angle to extract an image from the
+simulator. All custom pose extractors must inherit from the PoseExtractor abstract class and
+override the extract_poses method. Further, you must register the pose extractor using
+habitat_sim.registry.
```

Comment:
Sounds like your code below already handles the registration, right? If so, say so. Basically, so the user knows that they just need to keep that part of the code. 

Also, can you link to something sensible here so someone who wants to know more about the Habitat registry can go look up more.